tree/ntuple/v7/inc/ROOT/RNTupleDescriptor.hxx

```diff
@@ -359,16 +359,24 @@ public:
       };
 
       struct RPageInfoExtended : RPageInfo {
+      private:
          /// Index (in cluster) of the first element in page.
          ROOT::NTupleSize_t fFirstInPage = 0;
```

Comment:
I think this name is somewhat ambiguous, I would maybe change it to `fFirstElementIndex` (I think it should be clear that it refers to the page at hand, this is documented as well), together with the getters/setters.